src/main/java/com/flightstats/hub/dao/aws/S3WriteQueue.java

```diff
@@ -59,6 +68,10 @@ private S3WriteQueue() throws InterruptedException {
     private void write() throws InterruptedException {
         try {
             ChannelContentKey key = keys.poll(5, TimeUnit.SECONDS);
+            if (key != null) {
+                metricsService.gauge("s3.writeQueue.used", keys.size());
```

Comment:
Duplicated string, might deserve a constant?

Reply:
I thought about it but I went with consistency for now.